apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/ShardDownloadPresenceObserver.java

```diff
@@ -75,8 +76,21 @@ public ShardDownloadPresenceObserver(DatabaseManager databaseManager, Blockchain
      *
      * @param shardPresentData shard present data contains downloaded ZIP name
      */
-    public void onShardPresent(@Observes @ShardPresentEvent(ShardPresentEventType.SHARD_PRESENT) ShardPresentData shardPresentData) {
-        shardImporter.importShard(shardPresentData.getFileIdValue(), List.of());
+    public void onShardPresent(@ObservesAsync @ShardPresentEvent(ShardPresentEventType.SHARD_PRESENT) ShardPresentData shardPresentData) {
+        try {
```

Comment:
Why we should wait here?

Reply:
It gives ability to unpack zip successfully. Looks like it works fine now for me locally.

Reply:
Need to find correct solution. Using Thread.sleep - is not acceptable method of synchronization, random aspect is too big

Reply:
Already fixed